tap_zendesk/schemas/talk_calls.json

```diff
@@ -0,0 +1,187 @@
+{
+    "type": ["object"],
+    "properties": {
+        "agent_id": {
+            "type": ["integer", "null"],
+            "description": "The id of the first agent who picked up the call"
+        },
+        "call_charge": {
+            "type": ["string", "null"],
+            "description": "Total charge for the call. String representation of a decimal number with six decimal places. Example: \"1.230000\". Null if no charge was received from Twilio"
+        },
+        "call_group_id": {
+            "type": ["integer", "null"],
+            "description": "The id number of the group the call was last placed in before completion."
+        },
+        "call_recording_consent": {
+            "type": ["string", "null"],
+            "description": "Call recording consent value configured for the phone number",
+            "enum": ["always", "opt_in", "opt_out", "never"]
+        },
+        "call_recording_consent_action": {
+            "type": ["string", "null"],
+            "description": "Keypress the caller chose to give their call recording consent option.",
+            "enum": ["3"]
+        },
+        "callback": {
+            "type": ["boolean"],
+            "description": "True if the call was initiated by a callback request from the customer"
+        },
+        "callback_source": {
+            "type": ["string", "null"],
+            "description": "The source of the callback request",
+            "enum": ["queue", "web widget"]
+        },
+        "completion_status": {
+            "type": ["string", "null"],
+            "description": "Status of the call",
+            "enum": ["completed", "abandoned_in_queue", "abandoned_in_ivr", "abandoned_in_voicemail", "abandoned_on_hold", "pending_voicemail"]
+        },
+        "consultation_time": {
+            "type": ["integer"],
+            "description": "Sum of how long in seconds agents consulted with each other while the customer was on hold"
+        },
+        "created_at": {
+            "type": ["string"],
+            "description": "When the call object was created",
+            "format": "date-time"
+        },
+        "customer_id": {
+            "type": ["integer", "null"],
+            "description": "!!! not documented field !!!"
+        },
+        "customer_requested_voicemail": {
+            "type": ["boolean"],
+            "description": "The customer requested to be directed to voicemail instead of waiting for an agent to answer"
+        },
+        "default_group": {
+            "type": ["boolean"],
+            "description": "The call was answered by an agent who is a member of the call's default group, if group routing is used"
+        },
+        "direction": {
+            "type": ["string"],
```

Comment:
Please add "null" to handle the empty value. Do add null at all places.

Reply:
Done